install/helm/agones/templates/service/allocation.yaml

```diff
@@ -161,8 +184,19 @@ spec:
         - name: FEATURE_GATES
           value: {{ .Values.agones.featureGates | quote }}
         ports:
+        {{- if .Values.agones.allocator.service.http.enabled }}
         - name: https
-          containerPort: 8443
+          containerPort: {{ .Values.agones.allocator.service.http.targetPort }}
+        {{- if .Values.agones.allocator.service.grpc.enabled }}
+        {{- if ne .Values.agones.allocator.service.grpc.port .Values.agones.allocator.service.http.port }}
+        - name: grpc
+          containerPort: {{ .Values.agones.allocator.service.grpc.targetPort }}
+        {{- end }}
+        {{- end }}
+        {{- else if .Values.agones.allocator.service.grpc.enabled }}
+        - name: grpc
+          containerPort: {{ .Values.agones.allocator.service.grpc.targetPort }}
+        {{- end }}
         volumeMounts:
         - mountPath: /home/allocator/tls
```

Comment:
While it may not be directly related to this PR, it's worth noting that if you do have `disableTLS` or `disableMTLS` set to false, the `volumeMount` and corresponding `volume` should be gated off that value as well.

Reply:
I'll do this in a follow up PR to keep this change more focused.

Reply:
I've created https://github.com/googleforgames/agones/issues/2276 to track this feature request.